generate_sharktank.py

```diff
@@ -0,0 +1,80 @@
+# Lint as: python3
+"""SHARK Tank"""
+
+import os
+import urllib.request
+import csv
+import argparse
+import iree.compiler.tflite as ireec_tflite
+from shark.iree_utils import IREE_TARGET_MAP
+
+class SharkTank:
+
+    def __init__(self,
+                 torch_model_list: str = None,
+                 tf_model_list: str = None,
+                 tflite_model_list: str = None,
+                 upload: bool = False):
+        self.torch_model_list = torch_model_list
+        self.tf_model_list = tf_model_list
+        self.tflite_model_list = tflite_model_list
+        self.upload = upload
+
+        if self.torch_model_list is not None:
+            print("Process torch model")
+        if self.tf_model_list is not None:
+            print("Process torch model")
```

Comment:
Can we add a "TORCH HUB NOT IMPLEMENTED YET.", and "TF HUB NOT IMPLEMENTED YET."